clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1256,34 +1259,23 @@ def create_file(image, modality, bids_dir, mod_to_update):
         return nan
 
 
-def viscode_to_session(viscode):
-    """Replace the session label 'bl' with 'M00' or capitalize the session name passed as input.
-
-    Args:
-        viscode: session name
+def session_to_viscode(session_name: str) -> str:
```

Comment:
We have got a `session_to_viscode` and `viscode_to_session` but they don't treat the session the same way. They should be renamed appropriately to avoid any ambiguity:

`session_label_to_viscode(session_label: str) -> str` and `viscode_to_session_id(viscode: str) -> str`

Reply:
As a reminder, `session_id = f"ses-{session_label}"` as per the BIDS taxonomy.